source/templates/actions.md

```diff
@@ -255,3 +255,41 @@ will trigger the action, but the user will remain on the current page.
 
 With `preventDefault=false`, if the user clicked on the link, Ember.js
 will trigger the action *and* the user will be directed to the new page.
+
+### Modifying the action's first parameter
+
+If a `value` option for the `{{action}}` helper is specified, its value will be
+considered a property path that will be read off of the first parameter of the
+action. This comes very handy with event listeners and enables to work with
+one-way bindings.
+
+```handlebars
+<label>What's your favorite band?</label>
+<input type="text" value={{favoriteBand}} onblur={{action "bandDidChange"}}/>
```

Comment:
If you type something in the input box, `rating` does not get updated right away to the inputted value, only when you the input loses focus (in other words, when the action is fired). It is somewhat hard to get one's head around but maybe because we're too used to two-way bindings.